src/main/java/roomescape/member/infra/MemberRepository.java

```diff
@@ -30,7 +30,7 @@ public Long findIdByEmail(final String email) {
 
     public String findNameById(final Long id) {
         return MemberStorage.values().stream()
-                .filter(m -> m.getId().equals(id))
+                .filter(savedMember -> savedMember.getId().equals(id))
```

Comment:
```suggestion
                .filter(savedMember -> savedMember.hasId(id))
```
이런식으로 멤버가 직접 확인하게 해보는건 어떨까요?